### PR TITLE
Ft(Create Accomodation): Travel Admin creates accommodation facility 

### PR DESCRIPTION
**Pull request by tvpeter**
#### What does this PR do?

- Creates a functionality for `travel admin` to create accommodation facility

#### Description of Task to be completed?

- Check that a travel admin belongs to the Company to create accommodation facilities
- Checks that the accommodation is not created already
- Creates an accommodation facility with image
- Add rooms with their names and types

#### How should this be manually tested?

- run `npm i` to install the dependencies after cloning the branch

- start the dev server and visit `/api/v1/admin/accomodation/create` to create accomodation after logging in
- after creating accommodation, add rooms by creating a room by visiting `/api/v1/admin/accomodation/addroom` to add a room

#### Any background context you want to provide?
- none
#### What are the relevant pivotal tracker stories?

[#167891598](https://www.pivotaltracker.com/story/show/167891598)

#### Screenshots (if appropriate)

`creating accomodation`
Screen Shot 2019-09-08 at 7 24 31 AM
=====================
`creating rooms`
Screen Shot 2019-09-08 at 7 29 01 AM

#### Questions:

### CODE REVIEW

File: src/helpers/Token.js [hunk 1]
```diff
@@ -91,4 +91,4 @@ class Token {
   }
 }
 
-export default Token;
+export default Token;
```
Comment (Pomile) [inline src/helpers/Token.js]: I think there should be a new line after `export default Token`;

**Review by Pomile — APPROVED** [review]
Nice work!

File: src/database/seeders/20190821112700-users.js [hunk 2]
```diff
@@ -85,7 +85,31 @@ module.exports = {
...
+      password: hashHelper.hashPassword('password2019'),
+      gender: 'male',
+      dob: '2012-09-10',
+      status: 'active',
+      role: 'travel admin'
     }
   ]),
   down: queryInterface => queryInterface.bulkDelete('Users', null, {})
-};
+};
```
Comment (thatkazuk1) [inline src/database/seeders/20190821112700-users.js]: Newline here

**Review by thatkazuk1 — APPROVED** [review]
👍

**Review by max-wel — APPROVED** [review]

**Review by daniellamarr — APPROVED** [review]

**Pull request merged**